lib/image_processor/preprocessor/opencv-photo2scan.js

```diff
@@ -51,6 +53,7 @@ function getPaperThreshold(im, outfile) {
     mask.erode(5);
     mask.dilate(20);
 
+    /* istanbul ignore if */
```

Comment:
Is this the only place Istanbul ignore should be? There's plenty other `config.verbose` location.